### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by Feliverse**
**Learning objectives**

1. Use React Router.
2. Use styles in a React app.

**Description**
In this project, I create a full website for the Math magicians app, consisting of several pages and using the components already created.

**General requirements**

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- JavaScript requirements
- Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
- Project requirements
- In previous projects, you created the core functionality of the app: a Calculator component. Now you should develop a full website with several pages.
- 
- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- The general layout should match these wireframes:
- Core elements Core elements Core elements
- 
- Add your own styles to improve the look and feel.
- Use React basic styling (inline styles, importing CSS and pre-processed CSS files).
- For every page, you should create a <Route>.
- Make sure that navigation links for all 3 routes are displayed on each page.
- Use [React Router](https://reactrouter.com/) to make the routes work.

**Need a big picture?**
Remind me about [the big picture of this project](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/sneak_peek.md).

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
-    </section>
+    <div>
+      <Routes>
+        <Route path="/" element={<Layout />}>
+          <Route index element={<Home />} />
+          <Route path="calculator" element={<Calculator />} />
+          <Route path="quote" element={<Quote />} />
+        </Route>
+      </Routes>
+    </div>
```
Comment (VICTOR-CHUKWU) [inline src/App.js:18]: - [x] Good use of react router :clap:

File: src/index.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 import ReactDOM from 'react-dom/client';
 import './index.css';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
+  <BrowserRouter>
```
Comment (VICTOR-CHUKWU) [inline src/index.js:10]: - [x] [Optiona]try cleaning up your index.js file by removing the comments on this file :+1:

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Feliverse , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (Feliverse) [reply src/index.js:10]: Hi @VICTOR-CHUKWU , thanks for the review, I will follow your sugestions

**Pull request merged**